### DISCUSSION TITLE
Teacher connecting from other machine

### DISCUSSION THREAD

**Discussion by davidroberthoare** [epoptes/epoptes#197] [Q&A]
Hello :)
I've not got a small lab with a few clients connecting successfully to our server. However, we have multiple teachers who use the lab, and they bring their own laptops. Perhaps I'm misunderstanding the intended setup, but can a teacher somehow log-in or connect to the Epoptes server without being ON the server? We are not running LTSP; just stand-alone Epoptes. Do they have to VNC or remote login? or is there something built into the Epoptes server package that would allow them to run it on their local laptops and somehow connect to a 'main' server...?
Thank you
David

**Comment by alkisg** [comment]
Hi David,

up until the previous epoptes version, there were only these solutions: https://epoptes.org/documentation/run-fat/

While in the latest version, it's enough if they can ssh to the epoptes-server **passwordlessly**. That part is automated with LTSP, but if you configure it for non-LTSP installations, it'll work there as well.

I.e. the following should work without a password:

```
. /etc/default/epoptes-client
ssh $SERVER
```

...and their account should be a member of the `epoptes` group remotely on the epoptes server.

**Comment by an0nyme20G1rl** [comment]
hello, i need a person to teach me hacking, please i really need it, i would like to master the art of hacking, i am fluent in french and arabic and i speak english too , i am a student and i really need someone to teach me.
really I'm a beginner, I don't know anything, but I would like a little help, I was going to give up but a word inspired me "it's never too late" I want to learn that's all I ask.
  and thank u
